src/jit/lower.h

```diff
@@ -55,27 +55,63 @@ class Lowering : public Phase
             LowerNode(newNode);
         }
     }
+    void LowerRange(GenTree* firstNode, GenTree* lastNode)
+    {
+        for (GenTree* node = firstNode; node != lastNode->gtNext; node = node->gtNext)
+        {
+            LowerNode(node);
+        }
+    }
 
 private:
+    // ContainCheckRange handles new code that is introduced by or after Lowering,
+    // and that is known to be already in Lowered form.
+    void ContainCheckRange(LIR::Range&& range)
+    {
+        for (GenTree* newNode : range)
+        {
+            ContainCheckNode(newNode);
+        }
+    }
+    void ContainCheckRange(GenTree* firstNode, GenTree* lastNode)
+    {
+        for (GenTree* node = firstNode; node != lastNode->gtNext; node = node->gtNext)
+        {
+            ContainCheckNode(node);
+        }
+    }
+
+    void InsertTreeAndContainCheck(GenTree* insertionPoint, GenTree* tree)
```

Comment:
Done.

---
In reply to: [132026843](https://github.com/dotnet/coreclr/pull/13198#discussion_r132026843) [](ancestors = 132026843)